script/check-code-style.js

```diff
@@ -118,22 +118,31 @@ tests().forEach(file => {
   }
 });
 
-const isIgnoredForInterfaceCheck = (file) => {
-  return ['applicationmetadata'].includes(file);
+const isDirIgnoredForInterfaceCheck = (dir) => {
+  return ['applicationmetadata'].includes(dir);
 };
 
-components().forEach(component => {
-  if (isIgnored(component)) {
+const isFileIgnoredForInterfaceCheck = (file) => {
```

Comment:
Add this one to exclude the interface checking for specific files instead of excluding the whole directory.